lib/hterm.js

```diff
@@ -101,6 +101,23 @@ hterm.Keyboard.prototype.onKeyDown_ = function (e) {
     console.warn('Uncaught dead key on international keyboard', e);
   }
 
+  const modifierKeysConf = global.config.getConfig().modifierKeys;
+  if (e.altKey &&
+      e.code !== 'altLeft' &&
+      e.code !== 'altRight' &&
```

Comment:
Checking `e.code` because I don't want to send the alt KeyCode to the terminal. It would result to strange characters output in hyperterm.

Same on line 115 for the MetaKey